# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/redis/use-systemlib.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Source https://github.com/archlinux/svntogit-community/blob/packages/redis/trunk/redis-5.0-use-system-jemalloc.patch

diff --git a/src/Makefile b/src/Makefile
index 3bc9f11c0..a4b23d986 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -229,8 +229,8 @@ endif
 
 ifeq ($(MALLOC),jemalloc)
 	DEPENDENCY_TARGETS+= jemalloc
-	FINAL_CFLAGS+= -DUSE_JEMALLOC -I../deps/jemalloc/include
-	FINAL_LIBS := ../deps/jemalloc/lib/libjemalloc.a $(FINAL_LIBS)
+	FINAL_CFLAGS+= -DUSE_JEMALLOC -DJEMALLOC_NO_DEMANGLE  -I/usr/include/jemalloc
+	FINAL_LIBS := -ljemalloc -L/usr/lib64 $(FINAL_LIBS)
 endif
 
 ifeq ($(BUILD_TLS),yes)
